src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.ts

```diff
@@ -126,7 +161,10 @@ export const getProceedingDetails = (respondent: Respondent, req: AppRequest): P
 };
 
 function getLocalDate(orderDate: string): Date {
-  if (orderDate) {
+  if (orderDate['year'] === '' && orderDate['month'] === '' && orderDate['day'] === '') {
```

Comment:
Also this if else block can be written much simpler with short syntax

Reply:
Yes Vivek , This has been taken care .. the unnecessary else which was present before is now removed.